schedview/app/scheduler_dashboard/scheduler_dashboard.py

```diff
@@ -73,6 +73,11 @@
 """
 
 
+def get_color_palettes():
```

Comment:
Consider either indicating that this private (preceded by and underscore), or adding a docstring (if it is not).